### PR TITLE
Sankey chart - component tests

### PR DESCRIPTION
**Pull request by v-baambati**

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/a9fea90968d4fa7372fcd5b565d2fc578e02d424'>a9fea90968d4fa7372fcd5b565d2fc578e02d424</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/src/components/SankeyChart/SankeyChartRTL.test.tsx [hunk 1]
```diff
@@ -14,6 +61,62 @@ function sharedBeforeEach() {
...
+    container => {
+      // Assert
+      expect(container).toMatchSnapshot();
+    },
+  );
+});
+
+describe('Sankey chart - Subcomponent Node', () => {
+  testWithWait(
+    'Should update path color same as node clolor when we clck on node',
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/SankeyChart/SankeyChartRTL.test.tsx:78]: fix typo

**Review by AtishayMsft — APPROVED** [review]

**Review by AtishayMsft — CHANGES_REQUESTED** [review]
There is an error in the build pipeline.

Error: [console.error] The above error occurred in the <SankeyChartBase> component:
verb @fluentui/react-charting test |          
verb @fluentui/react-charting test |              at SankeyChartBase (/mnt/vss/_work/1/s/packages/react-charting/src/components/SankeyChart/SankeyChart.base.tsx:71:5)
verb @fluentui/react-charting test |              at /mnt/vss/_work/1/s/packages/utilities/src/styled.tsx:92:26

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**